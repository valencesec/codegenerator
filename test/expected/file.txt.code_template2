// file template: data file: "test/workdir/inline*.yaml" template file: "test/workdir/file.template"
// global data file: "test/workdir/globalData.yaml"
