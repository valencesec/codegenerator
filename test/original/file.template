capitalCamelCase:
{{- range $listItem := $.listOfItems }}
List Item: {{ $listItem }}
List Item quoted: {{ $listItem | quote }}
List Item capitalCamelCase: {{ $listItem | capitalCamelCase }}
{{- if $listItem | stringSliceContains $.listToCheckIn}}
List Item in listToCheckIn: {{ $listItem }}
{{- end }}
{{- if $listItem | stringSliceContains $.listToCheckIn | not}}
List Item not in listToCheckIn: {{ $listItem }}
{{- end }}
{{- end }}
HAS testing:
{{- if has . "testHas" }}
HAS works
{{- end }}
{{- if has . "testHasNot" }}
shloud not render
{{- end }}
