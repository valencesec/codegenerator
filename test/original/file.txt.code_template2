// file template: data file: "test/workdir/inline*.yaml" template file: "test/workdir/file.template"
