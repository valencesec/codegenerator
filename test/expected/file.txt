capitalCamelCase:
List Item: firstItem
List Item quoted: "firstItem"
List Item capitalCamelCase: FirstItem
List Item in listToCheckIn: firstItem
List Item: secondItem
List Item quoted: "secondItem"
List Item capitalCamelCase: SecondItem
List Item not in listToCheckIn: secondItem
HAS testing:
capitalCamelCase:
List Item: thirdItem
List Item quoted: "thirdItem"
List Item capitalCamelCase: ThirdItem
List Item not in listToCheckIn: thirdItem
List Item: fourthItem
List Item quoted: "fourthItem"
List Item capitalCamelCase: FourthItem
List Item in listToCheckIn: fourthItem
HAS testing:
HAS works
